src/components/background/Bubble.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+import { fly } from '../../style/animation';
+
+interface BubbleProps {
+  flagUrl?: string;
+  top?: string;
+  left?: string;
+  height?: string;
+  width?: string;
+  directions?: string;
+  zIndex?: number;
+  flyParams?: { x: number; y: number; time: number };
+}
```

Comment:
wszystkie parametry są opcjonalne? nie ma tak, ze niektóre muszą być przekazane? Jeśli tak to wtedy usuwamy `?`